nomination/urls.py

```diff
@@ -14,7 +14,7 @@
     path("help/", nomination_help, name='nomination_help'),
     path("<slug>/lookup/", url_lookup, name='url_lookup'),
     path("<slug>/search.json", search_json, name='search_json'),
-    path("<slug>/browse/<str>/browse.json", browse_json, name='browse_json'),
+    re_path(r"^([^/]+)/browse/([^/]+)/browse.json$", browse_json, name='browse_json'),
```

Comment:
Should it have been called "attribute", not "str"?

Reply:
I have used <slug> and <str> as path converters but they are being taken as keyword arguments. 
I just tried and checked, using - 
`path("<str:slug>/browse/<str:attribute>/browse.json", browse_json, name='browse_json'),` works fine too. 

I can use this to make sure we are replacing as many regexes in urls as possible.